harfbuzz/riscv64.patch

```diff
@@ -0,0 +1,22 @@
+--- PKGBUILD
++++ PKGBUILD
+@@ -14,8 +14,9 @@ makedepends=(glib2 freetype2 graphite cairo icu gobject-introspection gtk-doc
+ checkdepends=(python-fonttools python-setuptools)
+ options=(debug)
+ _commit=f1f2be776bcd994fa9262622e1a7098a066e5cf7  # tags/5.1.0^0
+-source=("git+https://github.com/harfbuzz/harfbuzz#commit=$_commit")
+-sha256sums=('SKIP')
++source=("git+https://github.com/harfbuzz/harfbuzz#commit=$_commit" "fix-char-signedness.patch::https://github.com/harfbuzz/harfbuzz/pull/3771.patch")
```

Comment:
It would be cleaner to place source URLs in separate lines.
```suggestion
+source=("git+https://github.com/harfbuzz/harfbuzz#commit=$_commit"
+        "fix-char-signedness.patch::https://github.com/harfbuzz/harfbuzz/pull/3771.patch")
```